documentation/asciidoc/computers/linux_kernel/about-kernel.adoc

```diff
@@ -1,12 +1,12 @@
 == Kernel
 
-The Raspberry Pi kernel is https://github.com/raspberrypi/linux[hosted on GitHub]; updates lag behind the upstream https://github.com/torvalds/linux[Linux kernel]. The upstream kernel is continuously updating; we take long-term releases of the kernel, which are mentioned on the front page, and integrate the changes into our own Raspberry Pi kernel. We then create a 'next' branch which contains an unstable port of the kernel, and after extensive testing and discussion, we push this to the main branch of our repository.
+The Raspberry Pi kernel is https://github.com/raspberrypi/linux[hosted on GitHub]; updates lag behind the upstream https://github.com/torvalds/linux[Linux kernel]. The upstream kernel updates continuously, whereas Raspberry Pi integrates **long-term releases** of the Linux kernel into the Raspberry Pi kernel. We generate a `next` branch in the Raspberry Pi kernel for each long-term Linux kernel release. After extensive testing and discussion, we merge each `next` branch into the main branch of our repository.
```

Comment:
I _think_ the `next` branch being discussed here is referring to https://github.com/raspberrypi/firmware/branches/all and https://github.com/raspberrypi/rpi-firmware/branches/all (i.e. as used by `sudo rpi-update next` https://github.com/raspberrypi/rpi-update/blob/master/README.md#options ) ? There's not a `next` branch in https://github.com/raspberrypi/linux/branches/all

(I believe https://github.com/raspberrypi/rpi-firmware contains binary-files created from the sourcecode and binaries in both https://github.com/raspberrypi/firmware and https://github.com/raspberrypi/linux )

Reply:
Updated with a link to rpi-firmware instead of insinuating that the `next` branch is part of the Raspberry Pi kernel repo.